src/EnergyPlus/Plant/LoopSide.cc

```diff
@@ -1718,7 +1716,7 @@ namespace DataPlant {
                                                                         FirstHVACIteration,
                                                                         LoopShutDownFlag,
                                                                         LoadDistributionWasPerformed);
-                    branch.Comp(CompCounter).simulate(state, FirstHVACIteration, DummyInit, DoNotGetCompSizFac);
+                    branch.Comp(CompCounter).simulate(state, FirstHVACIteration);
```

Comment:
And just as a note, this is the type of streamlining/performance improvement that won't show up as a hot spot in a profiler.  This is just knowing that every single call into this function will add a tiny delay because of a passive design, and cleaning it up so that it avoids that delay.  Great stuff.

Reply:
Got it. Thank you!